setup.py

```diff
@@ -118,17 +118,7 @@ def run(self):
       author='Roberto Alsina and others',
       author_email='ralsina@netmanagers.com.ar',
       url='https://getnikola.com/',
-      packages=['nikola',
-                'nikola.plugins',
-                'nikola.plugins.command',
-                'nikola.plugins.compile',
-                'nikola.plugins.compile.ipynb',
-                'nikola.plugins.compile.markdown',
-                'nikola.plugins.compile.rest',
-                'nikola.plugins.task',
-                'nikola.plugins.task.sitemap',
-                'nikola.plugins.template',
-                ],
+      packages=find_packages(),
```

Comment:
```py
      packages=find_packages(exclude=('tests',)),
```

Also, in the future, please work on a branch in getnikola/nikola (not your own fork) so we can collaborate.